sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1110,6 +1110,45 @@ pi_result piGetDeviceAndHostTimer(pi_device Device, uint64_t *DeviceTime,
   return pi2ur::piGetDeviceAndHostTimer(Device, DeviceTime, HostTime);
 }
 
+pi_result piextEnablePeerAccess(pi_device command_device,
+                                pi_device peer_device) {
+
+  std::ignore = command_device;
+  std::ignore = peer_device;
+
+  setErrorMessage("piextEnablePeerAccess not "
+                  "implemented in L0",
+                  UR_RESULT_ERROR_INVALID_VALUE);
+
+  return PI_ERROR_PLUGIN_SPECIFIC_ERROR;
+}
+
+pi_result piextDisablePeerAccess(pi_device command_device,
+                                 pi_device peer_device) {
+
+  std::ignore = command_device;
+  std::ignore = peer_device;
+
+  setErrorMessage("piextDisablePeerAccess not "
+                  "implemented in L0",
+                  UR_RESULT_ERROR_INVALID_VALUE);
+
+  return PI_ERROR_PLUGIN_SPECIFIC_ERROR;
+}
+
+pi_result piextPeerAccessGetInfo(pi_device command_device,
```

Comment:
L0 has zeDeviceGetP2PProperties, which returns caps for ACCESS and ATOMICS, is that what we want to return here?

Reply:
Yes exactly. I think it makes sense for someone who has access to a L0 gpu to do this in a follow up commit. I don't have access to a L0 machine. For now I plan to keep it as it is and simply return unsupported as suggested to me in above comments.

I just need to complete the above request to make a corresponding change in UR which I will get back to later today, then this PR can be merged (after resolving the latest conflicts).